.github/workflows/build.yml

```diff
@@ -48,6 +48,54 @@ jobs:
           CHANGED="$(EXTRA=build .github/files/list-changed-projects.sh)"
           echo "::set-output name=projects::${CHANGED}"
 
+      - name: Check if a WordPress.com test reminder comment exists
```

Comment:
Any particular reason for this to be a separate step instead of being merged with the next one?

Reply:
No particular reason. I just thought it was more readable that way.